web/src/components/tasks/business-formation/DbaFormationPaginator.tsx

```diff
@@ -0,0 +1,215 @@
+    nextStepIndex: number,
+    moveType: "NEXT_BUTTON" | "STEPPER"
+  ) => {
+    if (!formationFormData) {
+      return;
+    }
+
+    if (moveType === "STEPPER" && nextStepIndex == 0) {
+      analytics.event.business_formation_name_tab.click.arrive_on_business_formation_name_step();
+    }
+    if (moveType === "STEPPER" && nextStepIndex == 1) {
+      analytics.event.business_formation_dba_resolution_tab.click.arrive_on_business_formation_dba_resolution_step();
+    }
+    if (moveType === "STEPPER" && nextStepIndex == 2) {
+      analytics.event.business_formation_dba_authorization_tab.click.arrive_on_business_formation_dba_authorization_step();
+    }
+    if (moveType === "NEXT_BUTTON" && nextStepIndex == 1) {
+      analytics.event.business_formation_dba_resolution_step_continue_button.click.arrive_on_business_formation_dba_resolution_step();
+    }
+    if (moveType === "NEXT_BUTTON" && nextStepIndex == 2) {
+      analytics.event.business_formation_dba_authorization_step_continue_button.click.arrive_on_business_formation_dba_authorization_step();
+    }
+  };
+  const ForwardButton = () => {
+    const getForwardButtonText = () => {
+      if (isAuthenticated === IsAuthenticated.FALSE && isNotDba) {
+        return `Register & ${Config.businessFormationDefaults.initialNextButtonText}`;
+      } else {
+        return Config.nexusNameSearch.nexusNextButton;
+      }
+    };
+
+    return (
+      <Button
+        style="primary"
+        onClick={() => {
+          return onMoveToStep(state.stepIndex + 1, { moveType: "NEXT_BUTTON" });
+        }}
+        widthAutoOnMobile
+        noRightMargin
+        dataTestid="next-button"
+      >
+        {getForwardButtonText()}
+      </Button>
+    );
+  };
+
+  const BackButton = () => (
+    <Button style="secondary" widthAutoOnMobile onClick={onPreviousButtonClick}>
+      {Config.businessFormationDefaults.previousButtonText}
+    </Button>
+  );
+
+  const ButtonWrapper = (props: { children: ReactNode }) => {
+    return (
+      <div className="margin-top-2">
+        <div className="flex flex-justify-end bg-base-lightest margin-x-neg-4 padding-3 margin-top-3 margin-bottom-neg-4">
+          {props.children}
+        </div>
+      </div>
+    );
+  };
+
+  const displayButtons = () => {
+    if (state.stepIndex == 0 && state.businessNameAvailability?.status == "AVAILABLE") {
+      return (
+        <ButtonWrapper>
+          <ForwardButton />
+        </ButtonWrapper>
+      );
+    } else if (state.stepIndex == 1) {
+      return (
+        <ButtonWrapper>
+          <BackButton />
+          <ForwardButton />
+        </ButtonWrapper>
+      );
+    } else if (state.stepIndex == 2) {
+      return (
+        <TaskCTA
+          link={state.dbaContent.Authorize.callToActionLink}
+          text={state.dbaContent.Authorize.callToActionText}
+          onClick={() => {
+            return setShowCtaModal(true);
+          }}
+        >
+          <BackButton />
+        </TaskCTA>
```

Comment:
I think it feels weird to use the TaskCTA here... it has a specific purpose as a general Task component and it feels like putting it in a "special" component like this... would it be hard to just make a new button?